packages/cra-template/scripts/test.ts

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable no-console */
+import sh from '@fluentui/scripts/gulp/sh'; // eslint-disable-line
+import tmp from 'tmp'; // eslint-disable-line
+import path from 'path';
+
+// TODO this should ideally be merged with `packages/fluetui/project-tests/src/createReactApp.ts`
+
+// Clean up created files/folders on exit, even after exceptions
+// (will not catch SIGINT on windows)
+tmp.setGracefulCleanup();
+
+async function shEcho(cmd: string, cwd?: string) {
+  console.log(`+ cd ${cwd ?? '.'} && ${cmd}`);
+  await sh(cmd, cwd);
+}
+
+async function runE2ETest() {
+  try {
+    const tmpDir = tmp.dirSync({ prefix: 'cra-test-', unsafeCleanup: true });
```

Comment:
Can you make sure this test works on Windows (in cmd)?

Reply:
Will do. While we're here though, I thought we've had Win CI configured or am I missing something? Are we only running build+lint on linux? If so, perhaps it'd be worth to have all three major players run tests: linux, macOS and Win? I'd be happy to figure this out if this is something we'd want.

Reply:
I tested it in a Win VM and tests pass just fine.

Reply:
Thanks for checking! Currently we only build and do bundle tests on Windows (azure-pipelines.bundlesize.yml). That job is already pretty slow and a long tail of our build times, but we could potentially have it run tests as well if we split the build into multiple jobs (or just add an entirely separate pipeline which runs build/test/lint on Windows).